js/src/forum/index.js

```diff
@@ -64,7 +66,9 @@ app.initializers.add('flarum-markdown', function (app) {
         <MarkdownButton title={tooltip('header')} icon="fas fa-heading" onclick={makeApplyStyle('header')} />
         <MarkdownButton title={tooltip('bold', 'b')} icon="fas fa-bold" onclick={makeApplyStyle('bold')}  />
         <MarkdownButton title={tooltip('italic', 'i')} icon="fas fa-italic" onclick={makeApplyStyle('italic')}  />
+        <MarkdownButton title={tooltip('strikethrough')} icon="fas fa-strikethrough" onclick={makeApplyStyle('strikethrough')}  />
         <MarkdownButton title={tooltip('quote')} icon="fas fa-quote-left" onclick={makeApplyStyle('quote')} />
+        <MarkdownButton title={tooltip('spoiler')} icon="fas fa-minus-square" onclick={makeApplyStyle('spoiler')} />
```

Comment:
Personally, I'm more of a fan of https://fontawesome.com/v5.15/icons/eye-slash?style=solid

At the end of the day, it's personal preference, though. Let's get some more opinions.

/cc @tankerkiller125 @SychO9

Reply:
Hmmm, I kind of like the reddit Spoiler icon which is basically just an exclamation point.

Reply:
I do like the eye slash one, but it doesn't fit well when the preview button is an eye icon. So I'd probably go with the one Matt suggested.

Reply:
Yeah I'm okay with that. Any chance making changes on this PR for me?

Reply:
```suggestion
        <MarkdownButton title={tooltip('spoiler')} icon="fas fa-exclamation-triangle" onclick={makeApplyStyle('spoiler')} />
```